docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,442 @@
+# Working with Pascal VOC dataset from CLI
```

Comment:
Think of better placement for this text. The current variant looks a bit clumsy and unorganized. Consider adding a documentation section with tutorials or with format documentations.